### PR TITLE
[Fix] Added the Win32 not supported notice/warning

### PR DESCRIPTION
**Pull request by pedrominatel**
## Summary

Related to:
https://github.com/espressif/arduino-esp32/issues/6012
#5994
#5991

## Impact

None

### CODE REVIEW

File: docs/source/installing.rst [hunk 1]
```diff
@@ -61,6 +61,10 @@ To start the installation process using the Boards Managaer, follow these steps:
 Windows
 -------
 
+.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7 or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
```
Comment (atanisoft) [inline docs/source/installing.rst:64]: Windows 8.x should be listed here as well, there was another issue for Windows 8.1 x86 (32bit)

Comment (pedrominatel) [reply docs/source/installing.rst]: Thanks!

**Comment by VojtechBartoska** [comment]
other related issues added to the PR description.

File: docs/source/installing.rst [hunk 2]
```diff
@@ -61,7 +61,7 @@ To start the installation process using the Boards Managaer, follow these steps:
 Windows
 -------
 
-.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7 or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
+.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7, Windows 8.x x86 (32 bits) or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
```
Comment (VojtechBartoska) [inline docs/source/installing.rst:64]: Just small note, probably make more sence to say first Windows 8.x x86 (32 bits), Windows 7 or ealier..to keep timeline of Windows releases.

Some other thing is to probably add MacOS as well? Starting from MacOS 10.15, earlier versions are not supported. (I'm not a MacOS user though :) )

**Review by VojtechBartoska — APPROVED** [review]

**Pull request merged**